mosquitto/Dockerfile

```diff
@@ -4,36 +4,48 @@ FROM $BUILD_FROM
 # Install mosquitto + auth plugin
 WORKDIR /usr/src
 ARG MOSQUITTO_AUTH_VERSION
-RUN apk add --no-cache \
+RUN apt-get update \
+    && apt-get install -qy --no-install-recommends \
         mosquitto \
         nginx \
         pwgen \
-    && apk add --no-cache --virtual .build-dependencies \
-        build-base \
-        curl-dev \
+        build-essential \
         git \
         mosquitto-dev \
-        openssl-dev \
+        libmosquitto-dev \
+        openssl \
+        libssl-dev \
+        golang-go \
     \
     && git clone --depth 1 -b "${MOSQUITTO_AUTH_VERSION}" \
-        https://github.com/pvizeli/mosquitto-auth-plug \
+       https://github.com/iegomez/mosquitto-go-auth \
     \
-    && cd mosquitto-auth-plug \
-    && cp config.mk.in config.mk \
+    && cd mosquitto-go-auth \
+    && sed -i 's/-I\/usr\/local\/include/-I\/usr\/include/' Makefile \
+    && sed -i 's/LDFLAGS := .*$/& -Wl,-unresolved-symbols=ignore-all/' Makefile \
     && make \
     && mkdir -p /usr/share/mosquitto \
-    && cp -f auth-plug.so /usr/share/mosquitto \
-    && cp -f np /usr/local/bin \
+    && cp -f go-auth.so /usr/share/mosquitto \
+    && cp -f pw /usr/local/bin \
     \
-    && apk del --no-cache .build-dependencies \
+    && apt-get purge -y --auto-remove \
+        build-essential \
+        git \
+        mosquitto-dev \
+        libmosquitto-dev \
+        openssl \
+        libssl-dev \
+        golang-go \
```

Comment:
We have to cleanup the go compiler cache as well on the /root home

Reply:
So just `rm -rf /root/go` basically right? makes sense

Reply:
I think there is another dotCache folder on dotConfig or plain at home

Reply:
Found it, `/root/.cache`. Cleaned both.